stream_interfaces.go

```diff
@@ -69,13 +125,26 @@ type BidiStreamingClient[Req any, Res any] interface {
 // request message stream and the type of the response message stream. It is
 // used in generated code.
 type BidiStreamingServer[Req any, Res any] interface {
+	// Recv receives a request message from the client. The server-side handler can
+	// repeatedly call Recv on this parameter in order to read the client-to-server
+	// message stream. If an error occurs on the stream, it will be returned as an
+	// instance of the 'status' package. Refer to the 'status' package documentation
+	// for more details.Recv returns (nil, io.EOF) once it has reached the end of the
+	// client-to-server stream. when the client has finished sending messages.
```

Comment:
no need to say `client-to-server`. Interface documentation clarifies it is a server side stream.

Also, what is the purpose of `when the client has finished sending messages` sentence? Seems incomplete